pkg/test/framework/features/allowlist.txt

```diff
@@ -653,6 +653,9 @@ security,TestAuthorization_Deny
 security,TestAuthorization_EgressGateway
 security,TestAuthorization_GRPC
 security,TestAuthorization_IngressGateway
+security,TestAuthorization_IngressGatewayRemoteIp
```

Comment:
We don't need to change this as now the tests are in TestAuthorization_IngressGateway